tools/webdriver/webdriver/bidi/modules/storage.py

```diff
@@ -2,7 +2,10 @@
 
 from ._module import BidiModule, command
 
-from webdriver.bidi.modules.network import NetworkBytesValue
+from webdriver.bidi.modules.network import NetworkBytesValue, NetworkStringValue
+
+COOKIE_NAME = 'SOME_COOKIE_NAME'
+COOKIE_VALUE = NetworkStringValue('SOME_COOKIE_VALUE')
```

Comment:
It's questionable if we should use `NetworkStringValue` and change `recursive_compare`, or simply use a dict, copying the `NetworkStringValue` structure here, which would allow keep the `recursive_compare`.